data/evidenceLevels.json

```diff
@@ -452,6 +452,42 @@
             "sourceId": "e5",
             "url": "https://civicdb.org/glossary"
         },
+        "cosmic-1": {
```

Comment:
I'm just curious, why were these not already here? Cosmic isn't new to gkb is it?

Reply:
it isn't but the evidence levels are, they have recently released their actionability database. That send these are more aspirational at the moment since we are not loading that content yet. We are hoping future releases will have more applicable content in them